src/main/java/com/cinema/util/InjectData.java

```diff
@@ -0,0 +1,40 @@
+package com.cinema.util;
+
+import com.cinema.model.Role;
+import com.cinema.model.User;
+import com.cinema.service.RoleService;
+import com.cinema.service.ShoppingCartService;
+import com.cinema.service.UserService;
+import java.util.Set;
+import javax.annotation.PostConstruct;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class InjectData {
```

Comment:
```suggestion
public class DataInjector {
```
Class name should be a noun

Reply:
> Class name should be a noun
\+

Reply:
Done